.travis.yml

```diff
@@ -36,10 +36,11 @@ env:
     - JHIPSTER_LIB_BRANCH=release
     # if JHIPSTER_BRANCH value is release, use the release from NPM
     - JHIPSTER_REPO=https://github.com/jhipster/generator-jhipster.git
-    - JHIPSTER_BRANCH=v5.3.0
+    - JHIPSTER_BRANCH=v5.4.2
```

Comment:
This will definitely break things, The problem is after 5.3.0 a bunch of reactive stuffs were added. They are huge changes. 

I think we should just create a wrapper and ask people to use that instead of creating Kotlin files. That will simplify a lot here.

@jhipster/developers any ideas / feedbacks / suggestions welcome 👍